tools/ssm/linux-amd64-deb/install.sh

```diff
@@ -22,7 +22,9 @@ dpkg -i -E ./aws-otel-collector.deb
 readonly cmd='/opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl'
 readonly conf='/opt/aws/aws-otel-collector/etc/config.yaml'
 
-echo ${SSM_CONFIG} | base64 -d > ${conf}
+if [ ${SSM_CONFIG:+has_config} ]; then
```

Comment:
Where does `SSM_CONFIG` and `has_config` comes from? SSM parameter?

Reply:
SSM_CONFIG will be set to value of the SSM parameter by SSM Agent if there is Additional Arguments for SSM document. The "has_config" will be value of ${SSM_CONFIG:+has_config} when SSM_CONFIG is set and not Null. For detail explanation, please refer to https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02